### DISCUSSION TITLE
Cannot find training course - please help

### DISCUSSION THREAD

**Discussion by latlatina** [MIT-LCP/mimic-code#1563] [MIMIC-General]
Hi all,

Not a code question, but a technical one...
I read that in order to start using MIMIC, I have to go through training - "CITI Data or Specimens Only Research".
However, I cannot find this course on the website.
Did they change its name? Is it visible only to registered users?
Does anyone have the direct link to this course?

Thanks

**Comment by alistairewj** [comment]
It's a bit hidden, but there are step-by-step instructions here: https://physionet.org/about/citi-course/
